ml-agents/mlagents/trainers/tests/test_simple_rl.py

```diff
@@ -143,6 +144,7 @@ def _check_environment_trains(
         StatsReporter.add_writer(debug_writer)
         # Make sure threading is turned off for determinism
         trainer_config["threading"] = False
+        ghost_controller = GhostController()
```

Comment:
Does this need to be passed in as an arg? Looks like from the old code, the TrainerController could the instance from the TrainerFactory (but maybe I'm missing something).

Reply:
you aren't missing anything. I reverted that and its much cleaner